plugins/ltac/extraargs.mli

```diff
@@ -82,3 +82,5 @@ val wit_in_clause :
 val wit_strategy_level : Conv_oracle.level Genarg.uniform_genarg_type
 
 val wit_strategy_level_or_var : (Conv_oracle.level Locus.or_var, Conv_oracle.level Locus.or_var, Conv_oracle.level) Genarg.genarg_type
+
+val wit_binders : Constrexpr.local_binder_expr list Genarg.uniform_genarg_type
```

Comment:
This is not a true generic argument, but a vernac one. Only the raw type is relevant, it would fail for any other use. Please set the glob and top types to Empty.t. As a side-note, we should really have a dedicated type for vernac genargs.

Reply:
Right!

In second thought, I'm actually wondering why we need a `GRAMMAR EXTEND` indirection. There should be a way to tell that `Pcoq.Constr.binders` is available for grammar extensions (even tactics) w/o having to use this "trick".

Reply:
You can call `Pcoq.register_grammar` directly.

Reply:
Ping.

Reply:
I'll need some time to treat this one, sorry. Not prioritary for 8.13 anyway.

Reply:
Done, and it is now trivial.